config/database.yml

```diff
@@ -18,7 +18,7 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *local_default
-  database: vita-min_test<%= ENV['TEST_ENV_NUMBER'] %>
+  database: vita-min_test<%= ENV['TEST_ENV_NUMBER'] == '1' ? '' : ENV['TEST_ENV_NUMBER'] %>
```

Comment:
Can you explain this? I am not sure what is going on here.

Reply:
Em informs me this is a revert to a previous change. It looks like this affects local dev testing only.